### ISSUE TITLE
Make Instruction for blog update

### ISSUE THREAD

**Issue by lapin7** [synereo/blog.synereo.com#10]
I would say that the editing of the summary in MD and the  submitting as pull requests to github would be one reward. i.e. 60 USD. There just have to more people who know how to do that submitting to github. I've put the last summary in _drafts. It is ready for reviewing here https://github.com/synereo/blog.synereo.com/blob/gh-pages/_drafts/2016-02-17-Hangout-20160217.md

From there it could be moved to _post and then submitted to the blog.

**Comment by lapin7** [comment]
Locally I've moved the latest Hangout Summary  from the _draft directory to the _site directory.
$ jekyll build --drafts

I didn't rename the file, bute I should have done that.
I didn't fix the topmost Jekyll tags.
Currently I'm previewing the post in KRAMDOWN because the GitHub editor only renders 'markdown'.
$ jekyll serve --drafts

Running Jekyll on a local machine needs cloning the blog.synereo.com site to a live test site for previewing and testing.

I would be nice if this could be put in a clear manual.

**Comment by ian-bloom** [comment]
@lapin7 

> "I would say that the editing of the summary in MD and the submitting as pull requests to github would be one reward. i.e. 60 USD. "

I disagree. The technical requirements are too high and will scare off potential scribes.

**<draft> Blog Post Submission Workflow**
1.   clone blog.synereo.com to a new project
2.   in the new project, create branch#1
3.   in branch#1, re-format the new blog post into Kramdown
4.   check image licensing and attribution (Creative Commons, MIT, etc.)
5.   host images (ie., at imgur.com)
6.   add YAML front matter
7.   rename the file to _posts/(date)-Synereo-Update.markdown
8.   create branch#2 from branch#1
9.   modify branch#2 CNAME and _config.yml
10. publish branch#2 on GitHub as a website
11. visit newly published website to test formatting, links, pagination, etc.
12. submit a pull request on branch#1 referencing the link to newly published (dummy) site.

We can post the workflow to:
https://github.com/synereo/blog.synereo.com/wiki

**Comment by lapin7** [comment]
Yes, you're right.

So if the tech requirements are to high:
1. edit "Hangout yyyymmdd" with Googledocs and save it in folder: "Community Documents\Hangouts"
2. start the review process by a peer
3. translate into Spanish
4. start publishing proces for both below

I think we need to add the actual commands to the workflow or provide links to general manuals.
We also have to decide about the bounty for the publishing
The editing is now 60USD
The translation in Spanish 30USD
The publishing could be 10USD (>>>what do you think)

Great start of workflow description

**Comment by Elokane** [comment]
We need to set up a preview section so that people don’t have to clone the project to see the post.
